man/machine-id.xml

```diff
@@ -115,6 +116,34 @@
     early boot but become writable later on.</para>
   </refsect1>
 
+  <refsect1>
+    <title>First Boot Semantics</title>
+
+    <para><filename>/etc/machine-id</filename> is used to decide whether a boot is the first one.  The rules
+    are as follows:</para>
+
+    <orderedlist>
+      <listitem><para>If <filename>/etc/machine-id</filename> does not exist, this is a first boot.  During
+      early boot, systemd will create it with the content <literal>uninitialized</literal> and
```

Comment:
`unitialized\n` strictly speaking. Not sure if we want to clarify that...

This sentence is awkward. Maybe say:
`During early boot, systemd will write <literal>unitialized\n</literal> to this file and overmount a temporary file which contains the actual machine ID. Later (after <filename>first-boot-complete.target</filename> has been reached), the real machine ID will be written to disk.`

Reply:
> `unitialized\n` strictly speaking. Not sure if we want to clarify that...

Just for reference, the code will also work correctly when `\n` is missing.